components/server/src/test/java/com/hotels/styx/server/routing/AntlrMatcherTest.java

```diff
@@ -18,14 +18,16 @@
 import com.hotels.styx.server.HttpInterceptorContext;
 import org.junit.jupiter.api.Test;
 
+import java.util.concurrent.Executors;
+
 import static com.hotels.styx.api.LiveHttpRequest.get;
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
 
 public class AntlrMatcherTest {
 
-    private final HttpInterceptorContext contextHttps = new HttpInterceptorContext(true);
-    private final HttpInterceptorContext contextHttp = new HttpInterceptorContext();
+    private final HttpInterceptorContext contextHttps = new HttpInterceptorContext(true, null, Executors.newSingleThreadExecutor());
+    private final HttpInterceptorContext contextHttp = new HttpInterceptorContext(false, null, Executors.newSingleThreadExecutor());
```

Comment:
Should we replace this with Runnable:run as discussed? I don;t know if we can use requestContext()  here.

Reply:
👍